### PR TITLE
Remove IN_PROW annotation

### PR DESCRIPTION
**Pull request by jackdelahunt**
# Description
As the quota configuration is implemented, the IN_PROW annotation to be used in e2e tests is not needed any more.

[Ticket](https://issues.redhat.com/browse/MGDAPI-1625)

# Verify 
- Make sure all E2E tests that are ran in the openshift-ci pass on this pr.
- Verify changed tests pass
   - Provision a cluster and run RHOAM locally to install.
   - Run the following command `INSTALLATION_TYPE=managed-api TEST=A14 make test/e2e/single`
   - Then change the Quota of the install by setting the `QUOTA` environment variable and rerun the operator.
   - Run the A14 E2E test again to verify it passes. You can repeat this process using different quotas but setting the quota to 1 and 20 should be enough.
 - Add a `/lgtm` comment to this pr.

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist
- [ ] Verified independently on a cluster by reviewer

### CODE REVIEW

**Comment by MStokluska** [comment]
Hey @jackdelahunt, this will not work. By removing e2e/prow we would break our e2e tests. Currently, the way the e2e are run is via the make test/e2e/prow - this sets required env variables, and calls test/e2e. Test/e2e prepares the cluster for installation of RHOAM/RHMI and triggers the tests. 

This ticket is more about removing only the IN_PROW which ensures that the resources provisioned are at lower level due to the fact that the cluster provisioned is only 4node cluster (if I remember correctly).

File: Makefile [hunk 1]
```diff
@@ -214,7 +212,6 @@ test/e2e/rhoam/prow: export NAMESPACE_PREFIX := redhat-rhoam-
 test/e2e/rhoam/prow: export INSTALLATION_PREFIX := redhat-rhoam
 test/e2e/rhoam/prow: export INSTALLATION_NAME := rhoam
 test/e2e/rhoam/prow: export INSTALLATION_SHORTHAND := rhoam
-test/e2e/rhoam/prow: IN_PROW = "true"
```
Comment (austincunningham) [inline Makefile:217]: and this

File: Makefile [hunk 2]
```diff
@@ -201,7 +200,6 @@ test/e2e/prow: export NAMESPACE_PREFIX := redhat-rhmi-
 test/e2e/prow: export INSTALLATION_PREFIX := redhat-rhmi
 test/e2e/prow: export INSTALLATION_NAME := rhmi
 test/e2e/prow: export INSTALLATION_SHORTHAND := rhmi
-test/e2e/prow: IN_PROW = "true"
```
[the remark this answers is not in the document]
Comment (jackdelahunt) [reply Makefile:204]: The `IN_PROW` environment variable is only used in the operator for setting the desired replicas. Could it be that some of our pipelines check for this which is what is breaking the tests, I will add it back and see how the tests go.

**Pull request closed without merging**